### PR TITLE
Rename file and titles to use imperative.

### PR DESCRIPTION
**Pull request by osg-grafana**

### CODE REVIEW

File: docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 ---
-title: "Migrating from Thanos or Prometheus to Grafana Mimir"
-menuTitle: "Migrating from Thanos or Prometheus"
+aliases:
+  - migrating-from-thanos-or-prometheus/
```
Comment (osg-grafana) [inline docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: @jdbaldry, is this alias okay?

Comment (pracucci) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: I haven't manually checked it, but it's the same pattern we used for many other aliases where we just renamed the title.

**Review by pracucci — APPROVED** [review]

File: docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: You don't need an alias in this case because the page hasn't moved.

Comment (jdbaldry) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: An alias is a redirect from one page to another. We have some docs in https://grafana.com/docs/writers-toolkit/writing-guide/front-matter/#hugo-aliases.

Comment (pracucci) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: On right, the filename hasn't changed, but maybe Ursula meant to also change the filename? 🤔

Comment (jdbaldry) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: Probably, the PR title suggests that was intended.

Assuming the rename, the way I would check is to run the local webserver and browse to the old page location (http://localhost:3002/docs/mimir/latest/migrate/migrating-from-thanos-or-prometheus/). If the alias is correct, you should be redirected to the new page location.

Comment (jdbaldry) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: I wrote https://github.com/grafana/technical-documentation/blob/main/tools/cmd/link-checker/main.go to do this for me but I must admit I haven't run it in a while :D

Comment (jdbaldry) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: I think some assumptions about directory structure means that we can't yet use it here (https://github.com/grafana/technical-documentation/blob/main/tools/cmd/link-checker/main.go#L24, for example)

Comment (osg-grafana) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: > On right, the filename hasn't changed, but maybe Ursula meant to also change the filename? thinking

Yes, indeed, I meant to do that. heh

Comment (osg-grafana) [reply docs/sources/mimir/migrate/migrating-from-thanos-or-prometheus.md:3]: Classic result of multi-tasking. Thank you for catching that!

**Comment by osg-grafana** [comment]
I tested this locally now with the correct filename, and it works.

**Pull request merged**